velox/functions/lib/Re2Functions.cpp

```diff
@@ -416,12 +416,16 @@ bool matchSuffixPattern(
           length) == 0;
 }
 
+bool matchFixedCenterPattern(
+    const StringView& input,
+    const StringView& fixedPattern) {
+  return (strstr(input.data(), fixedPattern.data()) != nullptr);
```

Comment:
Is our StringView guaranteed to be null terminated for input?
I do not think so, the usage strstr might be reading memory outside the path